src/CommonITILTask.php

```diff
@@ -291,10 +291,55 @@ private function handleTaskDuration(array &$input, int $timestart, int $timeend)
         }
     }
 
+    public function assignTechFromtask($input)
+    {
+        //if user or group assigned to ticket task, add it to the ticket
+        $entities = [
+            'Ticket' => [
+                'users_id_tech' => new Ticket_User(),
+                'groups_id_tech' => new Group_Ticket()
```

Comment:
To get the correct xxx_user and group_xxx classes for an ITIL object, you have access to these properties:

![image](https://github.com/glpi-project/glpi/assets/42734840/faef08d6-8121-470b-b31f-f2fafa320028)